### PR TITLE
[14.0][MIG] event_registration_multi_qty: Migration to 14.0

### PR DESCRIPTION
**Pull request by chienandalu**
Standard migration

cc @Tecnativa TT34348

please review @sergio-teruel @ernestotejeda

### CODE REVIEW

File: event_registration_multi_qty/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from . import models
+from . import tests
```
Comment (ivantodorovich) [inline event_registration_multi_qty/__init__.py]: Shouldn't import tests here

**Review by ivantodorovich — APPROVED** [review]

File: event_registration_multi_qty/__init__.py (hunk 1 above)
Comment (chienandalu) [reply event_registration_multi_qty/__init__.py]: Thanks @ivantodorovich . Fixed :slightly_smiling_face::+1:

**Review by sergio-teruel — APPROVED** [review]
Code review

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**